### PR TITLE
i18n(es): Update `integrations-guides/prefetch` & `configuration-reference` and translate `guides/prefetch`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `integrations-guides/prefetch` & `configuration-reference` and translated `guides/prefetch`  based on the following commits:
- `guides/prefetch` => Added translation
- `integrations-guide`=> 63f1b9a9113a9fc178fede0500be02f5d466c04d
-  `configuration-reference`=> 0cbc0966c1974480cc5d09f6495fb80d878b81a5, 99a55e656ca7fd51cb44f4cd2fad6af2918214ff, 46bd0285df29a12101ec2ccd7e2a1dc17f82b276, f2a9c3b16956b326b64924ce4e8c59dc2560d320, 2a54003f1ab713295143a81c80723938d0e0d9fe & 06a01700be757bf4e65b55eebfb514d64fe0b122

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/prefetch.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+```js title="astro.config.js" ins={4}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  prefetch: true
+})
+```
+
+Un script de precarga se agregará a todas las páginas de su sitio. Luego puede agregar el atributo `data-astro-prefetch` a cualquier enlace `<a />` en su sitio para optar por la precarga. Cuando pasa el mouse sobre el enlace, el script buscará la página en segundo plano.
```
Comment (Waxer59) [inline src/content/docs/es/guides/prefetch.mdx:21]:
```suggestion
Un script de precarga se agregará a todas las páginas de tu sitio. Luego puedes agregar el atributo `data-astro-prefetch` a cualquier enlace `<a />` en tu sitio para optar por la precarga. Cuando pasa el cursor sobre el enlace, el script buscará la página en segundo plano.
```

File: src/content/docs/es/guides/prefetch.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+## Configuración de precarga
+
+La configuración `prefetch` también acepta un objeto de opciones para personalizar aún más la precarga.
+
+### Estrategias de precarga
+
+Astro admite 3 estrategias de precarga para varios casos de uso:
+
+- `hover` (default): Precarga cuando pasas el mouse o te enfocas en el enlace.
```
Comment (Waxer59) [inline src/content/docs/es/guides/prefetch.mdx:37]:
```suggestion
- `hover` (por defecto): Precarga cuando pasas el cursor o te enfocas en el enlace.
```

File: src/content/docs/es/guides/prefetch.mdx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+Puedes especificar una estrategia para un enlace individual pasándola al atributo `data-astro-prefetch`:
+
+```html
+<a href="/about" data-astro-prefetch="tap">Acerca</a>
+```
+
+Cada estrategia está ajustada para precargar solo cuando sea necesario y ahorrar el ancho de banda de tus usuarios. Por ejemplo:
+
+- Si un visitante está usando [data saver mode](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) o tiene una [conexión lenta](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType), la precarga se desactivará.
```
Comment (Waxer59) [inline src/content/docs/es/guides/prefetch.mdx:49]:
```suggestion
- Si un visitante está usando el [modo ahorro de datos](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) o tiene una [conexión lenta](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType), la precarga se desactivará.
```

File: src/content/docs/es/guides/prefetch.mdx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+  prefetch: {
+    prefetchAll: true
+  }
+})
+```
+
+Puedes optar por no precargar los enlaces individuales estableciendo `data-astro-prefetch="false"`:
+
+```html
+<a href="/about" data-astro-prefetch="false">About</a>
```
Comment (Waxer59) [inline src/content/docs/es/guides/prefetch.mdx:84]:
```suggestion
<a href="/about" data-astro-prefetch="false">Acerca</a>
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 5]
```diff
@@ -537,6 +524,71 @@ La opción de configuración de compilación `build.excludeMiddleware` ha sido r
...
+
+Habilita la precarga de enlaces en tu sitio para proporcionar transiciones de página más rápidas.
+(Activado de forma predeterminada en las páginas que utilizan el router `<ViewTransitions />`. Establece `prefetch: false` para optar por no utilizar este comportamiento.)
+
+Esta configuración agrega automáticamente un script de precarga a cada página del proyecto
+dandote acceso al atributo `data-astro-prefetch`.
+Agrega este atributo a cualquier enlace `<a />` en tu página para habilitar el prefetching para esa página.
+
+```html
+<a href="/about" data-astro-prefetch>About</a>
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:542]:
```suggestion
<a href="/about" data-astro-prefetch>Acerca</a>
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 6]
```diff
@@ -537,6 +524,71 @@ La opción de configuración de compilación `build.excludeMiddleware` ha sido r
...
+La estrategia de precarga predeterminada para utilizar cuando el atributo `data-astro-prefetch` está establecido en un enlace sin valor.
+
+- `'tap'`: Precarga justo antes de hacer clic en el enlace.
+- `'hover'`: Precarga cuando pasas el cursor o te enfocas en el enlace. (predeterminada)
+- `'viewport'`: Precarga cuando los enlaces entran en el viewport.
+
+Puedes reemplazar este valor predeterminado y seleccionar una estrategia diferente para cualquier enlace individual estableciendo un valor en el atributo.
+
+```html
+<a href="/about" data-astro-prefetch="viewport">About</a>
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:589]:
```suggestion
<a href="/about" data-astro-prefetch="viewport">Acerca</a>
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 7]
```diff
@@ -952,3 +994,109 @@ Habilita una superposición de desarrollo en el modo de desarrollo. Esta superpo
...
+**Por defecto:** 'prefix-other-locales'<br />
+<Since v="3.5.0" />
+</p>
+
+Controla la estrategia de enrutamiento para determinar las URL de tu sitio. Configura esto en función de la configuración de ruta de carpeta/URL para tu idioma predeterminado:
+
+ - `prefix-other-locales`(predeterminada): Solo los idiomas no predeterminados mostrarán un prefijo de idioma. 
+   El `defaultLocale` no mostrará un prefijo de idioma.
+   Los URLs serán de la forma `example.com/[lang]/content/` para todos los idiomas que no sean el predeterminado, pero `example.com/content/` para el idioma predeterminado.
+ - `prefix-always`: Todos los URLs mostrarán un prefijo de idioma.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1080]:
```suggestion
 - `prefix-always`: Todos las URLs mostrarán un prefijo de idioma.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 8]
```diff
@@ -952,3 +994,109 @@ Habilita una superposición de desarrollo en el modo de desarrollo. Esta superpo
...
+
+   Los URLs serán de la forma `example.com/[lang]/content/` para todas las rutas, incluido el idioma predeterminado.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1082]:
```suggestion
   Las URLs serán de la forma `example.com/[lang]/content/` para todas las rutas, incluido el idioma predeterminado.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**